handler/src/main/java/io/netty5/handler/ssl/SslCompletionEvent.java

```diff
@@ -15,25 +15,28 @@
  */
 package io.netty5.handler.ssl;
 
+import io.netty5.handler.codec.ProtocolEvent;
+
 import static java.util.Objects.requireNonNull;
 
-public abstract class SslCompletionEvent {
+public abstract class SslCompletionEvent<V> implements ProtocolEvent<V> {
```

Comment:
We could add some javadoc saying this event is sent when the TLS handshake completes.

Reply:
@chrisvest this is not true as its also used by the `SslClosedCompletionEvent`